zebra-chain/src/subtree.rs

```diff
@@ -50,61 +50,64 @@ impl From<NoteCommitmentSubtreeIndex> for u64 {
 // TODO:
 // - consider defining sapling::SubtreeRoot and orchard::SubtreeRoot types or type wrappers,
 //   to avoid type confusion between the leaf Node and subtree root types.
-// - rename the `Node` generic to `SubtreeRoot`
 
 /// Subtree root of Sapling or Orchard note commitment tree,
 /// with its associated block height and subtree index.
 #[derive(Copy, Clone, Debug, Eq, PartialEq)]
 #[cfg_attr(any(test, feature = "proptest-impl"), derive(Arbitrary))]
-pub struct NoteCommitmentSubtree<Node> {
+pub struct NoteCommitmentSubtree<Root> {
```

Comment:
Optional/Nitpick: Rename this to `SubtreeRoot` to avoid confusion with the normal `Root` types.

Reply:
Done in [a278777](https://github.com/ZcashFoundation/zebra/pull/7855/commits/a278777decfe04dfc265badd6c1049ab2697b83e).